docs/core/base/document.md

```diff
@@ -0,0 +1,112 @@
+# Document
+
+The term "Document" is used in the only in the code. It refers a single piece of content that created by Elementor, such as a post, page, or a saved template.
+
+Each document has its own type and features.
+
+A document is saved as a `post` in the WordPress `wp_posts` table. His elements and settings are saved as metadata in the `wp_postmeta` table.
+
+In PHP, the document is represented by the `Elementor\Base\Document` class.
+
+This class is like a DB Model that knows how to save and load its data.
+
+### Why "Document"?
+
+WordPress' content are called `posts` it's an historical term that came from the days of WordPress as a blogging system.
```

Comment:
```suggestion
In WordPress, contents are called `posts`. It's an historical term that was introduced in the days when WordPress was just a blogging system instead of the CMS it became to be today.
```

Maybe something like this